cmd/setup-etcd-environment/run.go

```diff
@@ -150,11 +156,16 @@ func reverseLookupSelf(service, proto, name, self string) (string, error) {
 	return selfTarget, nil
 }
 
-func writeEnvironmentFile(m map[string]string, w io.Writer) error {
+func writeEnvironmentFile(m map[string]string, w io.Writer, export bool) error {
 	var buffer bytes.Buffer
 	for k, v := range m {
 		buffer.WriteString(fmt.Sprintf("ETCD_%s=%s\n", k, v))
 	}
+	if export == true {
+		s := buffer.String()
+		buffer.Reset()
+		buffer.WriteString(fmt.Sprintf("set -a\n%sset +a\n", s))
```

Comment:
```
export <>
```
vs
```
set -a
<>
set +a
```
not sure why we picked one over the other.

Reply:
I like to enable/disable -a as a way to isolate a group of variables or an entire file of variables marked for export. If we were to add multiple vars with in the set it would not require any additional action. But I agree export is more common.

Reply:
will update which should simplify the code as well.